packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1329,14 +1329,22 @@ export class ContainerRuntime
 	 */
 	private nextSummaryNumber: number;
 
-	/** If false, loading or using a Tombstoned object should merely log, not fail */
+	/**
+	 * If false, loading or using a Tombstoned object should merely log, not fail.
+	 * @deprecated - This feature has been removed.
```

Comment:
Deprecated this for now. Didn't remove it because it would break legacy alpha APIs which is not allowed atm.

Reply:
Nit: `@deprecated` comments don't want a `-` between the tag and the body. Only `@param` blocks want that. As-is, the generated API docs for this would read like a single-entry list
```md
- This feature has been removed.
```

```suggestion
	 * @deprecated This feature has been removed.
```